app/src/main/java/com/pvp/app/ui/screen/goals/GoalScreen.kt

```diff
@@ -210,6 +234,87 @@ fun GoalCard(
     }
 }
 
+@Composable
+fun GoalCompletedCard(
+    goal: Goal,
+    monthSteps: Long
+) {
+    Box(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(vertical = 4.dp)
+            .clip(RoundedCornerShape(10.dp))
+            .background(MaterialTheme.colorScheme.surfaceContainer)
+            .border(
+                2.dp,
+                MaterialTheme.colorScheme.secondary,
+                RoundedCornerShape(10.dp)
+            )
+    ) {
+        Column(
+            modifier = Modifier
+                .padding(8.dp)
+                .fillMaxWidth()
+        ) {
+            Text(
+                style = MaterialTheme.typography.titleMedium,
+                modifier = Modifier.fillMaxWidth(),
+                textAlign = TextAlign.Center,
+                text = goal.activity.title + " goal is completed!"
+            )
+
+            Spacer(modifier = Modifier.padding(2.dp))
+
+            Text(
+                style = MaterialTheme.typography.labelSmall,
+                fontSize = 10.sp,
+                modifier = Modifier.fillMaxWidth(),
+                textAlign = TextAlign.Center,
+                text = goal.startDate.toString() + " - " + goal.endDate.toString()
+            )
+
+            Spacer(modifier = Modifier.padding(8.dp))
+
+            Row(modifier = Modifier.padding(6.dp)) {
+                Text(
+                    style = MaterialTheme.typography.bodyMedium,
+                    modifier = Modifier.padding(end = 8.dp),
+                    textAlign = TextAlign.Left,
+                    text = "Your goal was: ${
+                        when (goal.steps) {
+                            true -> "${goal.target.toInt()} steps"
+                            false -> "${goal.target} km"
+                        }
+                    }"
+                )
+
+                Icon(
+                    imageVector = goal.activity.icon,
+                    contentDescription = "Activity icon",
+                )
+            }
+
+            if (goal.steps) {
+                Spacer(modifier = Modifier.padding(2.dp))
+
+                Text(
+                    style = MaterialTheme.typography.bodyMedium,
+                    modifier = Modifier.padding(start = 6.dp),
+                    textAlign = TextAlign.Left,
+                    text = "Your average pace was " + when (goal.monthly) {
```

Comment:
I believe this is not really giving us average pace. We, most likely, want to show average pace in distance or steps **per day**. So if this was monthly goal, we divide sum by 30 or 31, or 28 (depends, unless we hard-coded it to 30 days when creating the goal). If it was weekly, we divide sum by 7.